### PR TITLE
fix(collector-create): fix the form bug of step 2

### PR DESCRIPTION
**Pull request by sulmoJ**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [x] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
- Initialize value of multipleProviderForm with first value.
- Add validation to provider form of step2.

### CODE REVIEW

File: apps/web/src/services/asset-inventory/collector/collector-create/modules/CreateCollectorStep2.vue [hunk 1]
```diff
@@ -8,6 +8,7 @@
             />
             <multiple-provider-form v-if="state.supportedProviders.length"
                                     class="multiple-provider-form"
+                                    @update:isProviderValid="handleChangeIsProviderValid"
```
Comment (yuda110) [inline apps/web/src/services/asset-inventory/collector/collector-create/modules/CreateCollectorStep2.vue:11]: nitpick: I think this is out convention...!

```
@update:is-provider-valid="handleChangeIsProviderValid"
```

**Review by yuda110 — APPROVED** [review]
LGTM!!

File: apps/web/src/services/asset-inventory/collector/collector-create/modules/MultipleProviderForm.vue [hunk 2]
```diff
@@ -56,6 +58,7 @@ import {
 const collectorFormStore = useCollectorFormStore();
 const collectorFormState = collectorFormStore.$state;
 
+const emit = defineEmits<{(event: 'update:is-provider-valid', value: boolean): void; }>();
```
Comment (piggggggggy) [inline apps/web/src/services/asset-inventory/collector/collector-create/modules/MultipleProviderForm.vue:61]: Suggestion: How about `update-valid` like something not having `update:-` prefix?
I think, using `update:-` appears to have some bound state.

**Review by piggggggggy — APPROVED** [review]
LGTM! I left a suggestion!

File: apps/web/src/services/asset-inventory/collector/collector-create/modules/MultipleProviderForm.vue (hunk 2 above)
Comment (sulmoJ) [reply apps/web/src/services/asset-inventory/collector/collector-create/modules/MultipleProviderForm.vue:61]: Oh. I love it.

**Pull request merged**